ui/webpack.config.js

```diff
@@ -17,9 +17,32 @@ module.exports = {
         use: {
           loader: 'babel-loader',
           options: {
-            presets: ['@babel/typescript', '@babel/preset-react', '@babel/preset-env']
+            presets: ['@babel/preset-env', '@babel/preset-react', '@babel/typescript']
           }
         }
+      },
+      {
+        test: /\.((c|sa|sc)ss)$/i,
+        use: [
+          'style-loader',
+          {
+            loader: 'css-loader',
+            options: {
+              importLoaders: 1,
+              modules: { auto: true }
```

Comment:
this line enable css-modules, but only recognize the suffix with `.module.(scss|css|...)`,so seems the original usage of `.scss` still be work